### DISCUSSION TITLE
Routing conflict when put page.js in folder's name in parenthesis

### DISCUSSION THREAD

**Discussion by phatify** [vercel/next.js#41947] [Help]
I found an new issue of define route in Next.js 13, I wonder whats going on if we put both of index routing (page.js) in two parenthesis folder and more? Why I accessing `http://localhost:3000/` from a browser it always return routing from (user) ?

Example: 

```
app/
   (home)
        page.js
   (user)
        page.js
   layout.js
```

**Comment by icyJoseph** [comment]
Groups do not affect the actual URL, from the docs https://beta.nextjs.org/docs/routing/defining-routes#example-creating-multiple-root-layouts:


> Good to know:
> 
> The naming of route groups has no special significance other than for organization. They do not affect the URL path.
> **Routes inside route groups should not resolve to the same URL path.** For example, since route groups don't affect the URL structure, the following combination would both **resolve to /about and cause an error: (marketing)/about/page.js and (shop)/about/page.js**

So your folder structure is defining two landing pages.

**Reply by phatify** [reply]
But I think Nextjs should show a message warning to the console, so it's easier for developers to control

**Comment by phatify** [comment]
@timneutkens hi bro
